src/PageContent/FoodLogs/FoodLogItem.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+import './FoodLogItem.css';
+class FoodLogItem extends Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            currentPage: 1,
+            foodPerPage: 4
+        };
+        this.handleClick = this.handleClick.bind(this);
+    }
+    handleClick(event){
+        this.setState({
+            currentPage: Number(event.target.id)
+        });
+    }
+    render(){
+        const { currentPage, foodPerPage } = this.state;
+        const foodItems = this.props.delivery.description.foodItems;
+
+        // Logic for displaying current foods
+        const indexOfLastFood = currentPage * foodPerPage;
+        const indexOfFirstFood = indexOfLastFood - foodPerPage;
+        const currentFoods = foodItems.slice(indexOfFirstFood, indexOfLastFood);
+
+        const renderFood = currentFoods.map((food, i) => {
+            return(
+                <div className="food" key={i}>
+                    <p className="info-detail donationItem">{food.food} &nbsp;</p>
+                    <p className="info-detail donationWeight">{food.quantity} {food.unit}</p>
+                </div>
+            );
+        });
+
+        // Logic for displaying page numbers
+        const pageNumbers = [];
+        for (let i = 1; i <= Math.ceil(foodItems.length / foodPerPage); i++) {
+            pageNumbers.push(i);
+        }
+        let renderPageNumbers = null;
+        if(pageNumbers.length > 1) {
+            renderPageNumbers = pageNumbers.map(number => {
+                return (
+                    <p className={number === this.state.currentPage ? 'circle': 'circle open' } key={number} id={number} onClick={this.handleClick}>
+                    </p>
+                );
+            });
+        }
+        return(
+            <div className="item">
+                <div className="heading">
+                    <p className="delivery">Delivery Complete &nbsp;</p> 
+                    <p className="date"> &nbsp; on &nbsp; {moment(this.props.delivery.deliveredInfo.timestamp).format('LLL')}</p>
+                </div>
+                <div className="flex">
+                    <div className="info ">
+                        <div className="subitem">
+                            <p className="info-title">Donating Agency</p>
+                            <p className="info-subtitle">{this.props.delivery.donatingAgency.name} ({this.props.delivery.daContact.phone})</p>
+                            <p className="info-detail">{this.props.delivery.daContact.name}</p>
+                            <p className="info-detail">Signed at {moment(this.props.delivery.pickedUpInfo.timestamp).format('LT')}</p>
+                            <p className="info-detail">{this.props.delivery.daContact.email}</p>
```

Comment:
lol I don't understand why the design is so inconsistent, but I feel like we should have the same as Receiving Agency info below?
```
Signed by: {delivery.pickupedInfo.signature} // show who actually signed instead of daContact
Timestamp: ...
```
I'm also not sure why we're showing email here, maybe ask Madison about this? Because whoever signed might not actually be the `daContact`/`raContact` that we have on file.

Reply:
I asked, no reply yet